code/game/objects/items/devices/traitordevices.dm

```diff
@@ -321,29 +361,48 @@ effective or pretty fucking useless.
 /obj/machinery/porta_turret/syndicate/toolbox/target(atom/movable/target)
 	if(!target)
 		return
+
 	if(shootAt(target))
 		setDir(get_dir(base, target))
+
 	return TRUE
 
 /obj/machinery/porta_turret/syndicate/toolbox/attackby(obj/item/attacking_item, mob/living/user, params)
-	if(istype(attacking_item, /obj/item/wrench/combat))
-		if(user.combat_mode && attacking_item.toolspeed && attacking_item.use_tool(src, user, 5 SECONDS, volume = 20))
-			deconstruct(TRUE)
-			attacking_item.play_tool_sound(src, 50)
-		else if(!user.combat_mode)
-			to_chat(user, span_notice("You start repairing [src]..."))
-			while(atom_integrity != max_integrity && attacking_item.toolspeed && attacking_item.use_tool(src, user, 2 SECONDS, volume = 20))
```

Comment:
should keep a loop in so you dont have to keep clicking it

Reply:
Coffee told me that balloon alert will break loop and that's there is no reason to use while as it's a if

Reply:
Also I kind of can't think how to add balloon alert to while ... Unless I change it into a loop without early return but I will do that in few hours as im currently busy